brave/browser/brave_content_browser_client.cc

```diff
@@ -282,6 +289,17 @@ void BraveContentBrowserClient::GetStoragePartitionConfigForSite(
   // error about which StoragePartition they expect to be in and it is not
   // safe to continue.
   CHECK(can_be_default || !partition_domain->empty());
+
+  auto brave_browser_context =
+    BraveBrowserContext::FromBrowserContext(browser_context);
+  if (brave_browser_context && brave_browser_context->IsIsolatedStorage() &&
+      !site.SchemeIs(extensions::kExtensionScheme)) {
+    if (!site.is_empty()) {
+      *partition_domain = site.host();
+      *partition_name = site.host();
+    }
+    *in_memory = true;
```

Comment:
this should be set to `brave_browser_context->IsOffTheRecord()` so we don't run into problems later if we add support for persistent tor tabs

Reply:
This is for the case of `IsIsolatedStorage` and it is always in memory https://github.com/brave/muon/pull/473/files#diff-df5c427fd693f9ebb2e5952cf0e9c8c7R388

Reply:
`IsIsolatedStorage` does not imply `in_memory` so I still think this should use `IsOffTheRecord`